### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by zowoq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,27 +4,38 @@ Welcome to the Nix Community infrastructure project. This project holds all the
 
 ## Services
 
-### Community builder - build01.nix-community.org
+### `Community builder` - build01.nix-community.org
 
 We also provide one x86 hetzner build machine as a public remote builder for the nix community, see [here](roles/builder/README.md) for more information.
```
Comment (zimbatm) [inline README.md:9]: Should we mention the aarch64 machine as well?

Comment (zowoq) [reply README.md]: Added aarch64 for the builder and CI systems.

**Review by zimbatm — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@ See [here](roles/builder/README.md) for more information.
 We provide x86 and aarch64 linux CI via these systems:
 
 - `Hercules` - https://hercules-ci.com/github/nix-community
+  - To enable hercules builds go to `https://hercules-ci.com/github/nix-community/$REPO` and click "Build this repository".
```
Comment (zowoq) [inline README.md:18]: For this need to disable new repos in  https://hercules-ci.com/accounts/github/nix-community/settings.

**Comment by zowoq** [comment]
> /bors r+

No `/`, just `bors $cmd`.

**Comment by Mic92** [comment]
bors merge

**Pull request merged**